### PR TITLE
Increases the price of automatic weapons and some handguns

### PR DESCRIPTION
**Pull request by Martinpachu**
## About The Pull Request

Self explanatory. Also slightly nerfs the price on pistols to make them more balanced compared to revolvers.

## Why It's Good For The Game

Automatics are EXTREMELY powerful and it's supposed to be a big deal to equip everyone or most people in a crew with them. ESPECIALLY AR's. Pistols one is more to just make revolvers more viable.

## Changelog

:cl:

balance: Doubled the price on every automatic gun and added 250 credits to the price of every semi-auto pistol. 
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs/gun.dm [hunk 1]
```diff
@@ -165,27 +165,27 @@
...
-	cost = 5000
+	cost = 10000
 	contains = list(/obj/item/gun/ballistic/automatic/assault/p16)
 	crate_name = "auto rifle crate"
 
 /datum/supply_pack/gun/skm
 	name = "SKM-24 Rifle Crate"
 	desc = "Contains a high-powered, automatic rifle chambered in 7.62x40mm CLIP."
-	cost = 5000
+	cost = 1000
```
Comment (generalthrax) [inline code/modules/cargo/packs/gun.dm]: Was it intended to make the SKM fives times less expensive than originally?

Comment (Martinpachu) [reply code/modules/cargo/packs/gun.dm]: Yeah, i really think it's an u i forgot ot push the fix

Comment (generalthrax) [reply code/modules/cargo/packs/gun.dm]: my boy

**Comment by generalthrax** [comment]
Might want to increase mech price while you're at it. iirc as it stands a full Gygax DIY kit is just 2000 more than a P16 or SKM

**Comment by Gristlebee** [comment]
> Might want to increase mech price while you're at it. iirc as it stands a full Gygax DIY kit is just 2000 more than a P16 or SKM

Yep, Gygax is 12k at the outpost, but you do still have to buy equipment for the mech as well, which is usually another couple thousand on average.

File: code/modules/cargo/packs/gun.dm [hunk 2]
```diff
@@ -165,27 +165,27 @@
...
-	cost = 5000
+	cost = 10000
 	contains = list(/obj/item/storage/guncase/p16)
 	crate_name = "auto rifle crate"
 
 /datum/supply_pack/gun/skm
 	name = "SKM-24 Rifle Crate"
 	desc = "Contains a high-powered, automatic rifle chambered in 7.62x40mm CLIP."
-	cost = 5000
+	cost = 1000
```
Comment (SomeguyManperson) [inline code/modules/cargo/packs/gun.dm]:
```suggestion
	cost = 10000
```

**Review by FalloutFalcon — APPROVED** [review]

**Pull request merged**